content/en/docs/gke/deploy/management-setup.md

```diff
@@ -39,39 +39,70 @@ For a more detailed explanation of the drastic changes happened in vKubeflow 1.1
 
     **Note:** Starting from Kubeflow v1.2, we fixed the compatibility problem with Kustomize `v3.2.1+`, so you can now install any Kustomize `v3+`, including the latest Kustomize versions.
 
+    To deploy the latest version of Kustomize on a Linux or Mac machine, run the following commands:
+
+    ```bash
+    # Detect your OS and download corresponding latest Kustomize binary
+    curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+
+    # We need to add the kustomize package to your $PATH env variable
+    sudo mv ./kustomize /usr/local/bin/kustomize
+    ```
+
+    Then, to verify the installation, run `kustomize version`. You should see `Version:kustomize/vX.Y.Z` in the output if you've successfully deployed Kustomize.
+
 1.  Install [yq](https://github.com/mikefarah/yq#install).
 
-## Setting up the management cluster
+## Setting up a management cluster
+
+This guide assumes the following convention:
+
+- The `${MGMT_DIR}` environment variable contains the path to
+  your management directory, which holds your management cluster configuration
+  files. For example, `~/kf-deployments/management/`. You can choose any path you would like for the directory `${MGMT_DIR}`.
+
+  You are recommended to check the management directory in source control.
+
+- The `${MGMT_NAME}` environment variable contains the name of your management cluster.
+
+However, the environment variables are used purely for command illustration purpose. No tools will assume they actually exists in your terminal environment.
+
+To deploy a management cluster:
 
 1.  Fetch the management blueprint to current directory
 
     ```bash
-    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 ./
+    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 "${MGMT_DIR}"
     ```
 
+1.  Change to the Kubeflow directory
+
+    ```bash
+    cd "${MGMT_DIR}"
+    ```
+
+    Note, all the instructions below assume your current working directory is `${MGMT_DIR}`.
+
 1.  Fetch the upstream management package
 
     ```bash
-    cd ./management
     make get-pkg
     ```
 
-    Note, all the instructions below assume your current working directory is `./management`.
-
 1.  Use kpt to set values for the name, project, and location of your management cluster:
 
     ```bash
-    kpt cfg set -R . name ${NAME}
-    kpt cfg set -R . gcloud.core.project ${PROJECT}
-    kpt cfg set -R . location ${LOCATION}
+    kpt cfg set -R . name "${MGMT_NAME}"
```

Comment:
As with other docs in this PR, please replace undefined environment variables that are used once with a placeholder, and then define the variable below the example.